conda-recipes/tensorflow-metadata-feedstock/ci_support/run_docker_build.sh

```diff
@@ -0,0 +1,74 @@
+# (C) Copyright IBM Corp. 2018, 2019. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/usr/bin/env bash
+
+THISDIR="$( cd "$( dirname "$0" )" >/dev/null && pwd )"
+PROVIDER_DIR="$(basename $THISDIR)"
+
+FEEDSTOCK_ROOT=$(cd "$(dirname "$0")/.."; pwd;)
+RECIPE_ROOT=$FEEDSTOCK_ROOT/recipe
+
+docker info
+
+config=$(cat <<CONDARC
```

Comment:
The condarc is created with the conda-forge tools now in the `build_steps` file. This `config=` here can be removed.